src/main/java/com/quizapp/enterprise/persistence/GameTracker.java

```diff
@@ -47,19 +47,19 @@ public void joinGame(
                 .add(playerToJoin);
     }
 
-    public Question getNextQuestion(String gameCode){
+    public Question getNextQuestion(String gameCode) throws Exception {
 
         Game game = getGameByCode(gameCode);
+      if(game == null){
+          throw new Exception("game cannot be null");
+      }
```

Comment:
included a check for a possible null prior to execution on possible null object so code may handle the case as desired.
This change was recommended by the "8. Proper handling of Null Pointer Exceptions"  section of  the java coding best practices summary referenced below

source:https://xperti.io/blogs/java-coding-best-practices/

Reply:
If you didn't do this, a NullPointerException is thrown.
If you do this, an Exception is thrown.  
Those are both equivalent. 

I'd consider ways to heal the null here, if possible, in lieu of throwing the exception.
Or, see if a null can be prevented in the first place.